wqflask/wqflask/snp_browser/snp_browser.py

```diff
@@ -645,6 +643,79 @@ def snp_density_map(self, query, results):
         #for i in range(n_click):
         #    href = url_for('snp_browser', first_run="false", chosen_strains_mouse=self.chosen_strains_mouse, chosen_strains_rat=self.chosen_strains_rat, variant=self.variant_type, species=self.species_name, gene_name=self.gene_name, chr=self.chr, start_mb=self.start_mb, end_mb=self.end_mb, limit_strains=self.limit_strains, domain=self.domain, function=self.function, criteria=self.criteria, score=self.score, diff_alleles=self.diff_alleles)
 
+class SnpPage(object):
+
+    def __init__(self, request):
```

Comment:
**issue:** @Uditgulati I'd recommend we rename the request object to something else. I'm not sure whether this is python's request(the import lib) object or not. If not, let us rename this to something else to make code easier to read. Also, I'll eventually run pylint on everything in future, so having one less thing to correct would be a big +1 :)

Reply:
I'll change it to something like `request_data` or `request_values`.